_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -3357,35 +3357,35 @@ sections above.
 1.1,<<security_questions>>,Complete the contact details on the AWS account page
 1.2,<<security_questions>>,Complete the security contact information on the AWS account page
 1.3,<<security_questions>>,Answer the security questions on the AWS account page
-1.4,<<next_steps>>,Use the Gruntwork Security Hub module to enable AWS Security Hub to ensure that no access key exists for the root user
+1.4,<<configure_security_hub>>,Use the Gruntwork Security Hub module to enable AWS Security Hub to ensure that no access key exists for the root user
```

Comment:
**_Context:_** Technically, I believe Security Hub isn't part of the CIS official requirements. It is, however, a useful tool to monitor that for example this is a fulfilled rule. It checks currently against CIS 1.2 benchmark, not even 1.3.

Reply:
True. If there's a better section to link to here, let me know.

Reply:
I think by adding the section for Security hub, this makes more sense now

Reply:
Yeah, it looks like it's required in CIS 1.4.0, so we'll probably be adding it in the future anyway.